media/js/firefox/firstrun/new-firstrun.js

```diff
@@ -9,8 +9,27 @@
         var scene = document.getElementById('scene');
         var skipbutton = document.getElementById('skip-button');
 
-        var hideSkipButton = function () {
-            skipbutton.style.display = 'none';
+        var hideOrShowSkipButton = function (data) {
+            switch(data.data.url) {
+            case 'signin':
+            case 'signup':
+            case 'reset_password':
+                skipbutton.removeClass('skipbutton-hidden');
```

Comment:
This throws an error, as `removeClass` is a jQuery function. You could either change `skipbutton` to a jQuery object (which would require a bit more of a refactor), *or*, since `firstrun` pages are basically guaranteed to run on a modern version of Firefox, you could use the [classList API](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList). (I'd go for the latter 😉.)